ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -174,6 +177,13 @@ data Peer a =
 instance Functor Peer where
   fmap f Peer {name, value} = Peer {name, value = f value}
 
+instance Foldable Peer where
+  foldr step z (Peer _ a) = step a z
+
+instance Traversable Peer where
+  sequenceA (Peer name fa) =
+    fa <&> Peer name
```

Comment:
Nice!

Reply:
forgot to change the `<&>` back to `<$>` though 😁